plugin/iamutil/iam_resource_test.go

```diff
@@ -218,7 +219,7 @@ func TestConditionalIamResource(t *testing.T) {
 				Condition: &Condition{
 					Title:       "test",
 					Description: "",
-					Expression: "a==b",
+					Expression:  "a==b",
```

Comment:
I think adding an extra whitespace here is a typo?

Reply:
Nope, that was added by the automatic go formatter.

Reply:
OK, thanks for clarifying :+1: